tools/build-templated-pages/src/features.rs

```diff
@@ -25,29 +25,9 @@ impl PartialOrd for Feature {
     }
 }
 
-fn parse_features(panic_on_missing: bool) -> Vec<Feature> {
-    let manifest_file = std::fs::read_to_string("Cargo.toml").unwrap();
-    let manifest = manifest_file.parse::<Document>().unwrap();
-
-    let features = manifest["features"].as_table().unwrap();
-    let default: Vec<_> = features
-        .get("default")
-        .unwrap()
-        .as_array()
-        .unwrap()
-        .iter()
-        .flat_map(|v| {
-            std::iter::once(v.as_str().unwrap().to_string()).chain(
-                features
-                    .get(v.as_str().unwrap())
-                    .unwrap()
-                    .as_array()
-                    .unwrap()
-                    .iter()
-                    .map(|v| v.as_str().unwrap().to_string()),
-            )
-        })
-        .collect();
+fn parse_features(what_to_run: Command) -> Vec<Feature> {
+    let features = get_manifest_features();
+    let default: Vec<_> = get_default_features(&features);
```

Comment:
```suggestion
    let default = get_default_features(&features);
```

The type is implied here, it can be removed.